source/ShipInfoDisplay.cpp

```diff
@@ -249,7 +250,7 @@ void ShipInfoDisplay::UpdateAttributes(const Ship &ship, const Depreciation &dep
 	}
 	
 	// Print the number of bays for each bay-type we have
-	for(auto bayType : Ship::BAY_TYPES)
+	for(auto bayType : GameData::Category(CategoryType::BAY_TYPE))
```

Comment:
```suggestion
	for(auto &&bayType : GameData::Category(CategoryType::BAY_TYPE))
```
Might as well fix the extra unconditional copy that was here.